aten/src/ATen/cuda/tunable/TunableOp.h

```diff
@@ -205,7 +237,9 @@ class TunableOp {
         }
       }
 
-      reusable_params->Delete();
+      for (size_t i = 0; i < reusable_params.size(); i++) {
+        reusable_params[i]->Delete();
+      }
       reference_params->Delete();
 
```

Comment:
Call 
```
gc.collect()
torch.cuda.empty_cache()
```
to force release CUDA memory here? Currently see some OOM issues when trying this PR on some internal workloads.

Reply:
The code you suggest are python APIs.  The reviewed code you highlighted is C++.  Using TunableOp does increase memory pressure during tuning, but once tuning is complete it should be similar memory pressure as before (with the exception of any changes you make to CUBLASLT_WORKSPACE_SIZE env var).  Disabling the numerics check (on by default) and the buffer rotation (off by default) if you don't need them will save some memory, as well.